### ISSUE TITLE
Issue with shortcodes displaying in Post Editor for v0.2.2

### ISSUE THREAD

**Issue by hirejordansmith** [wp-shortcake/shortcake#267]
I updated a few sites that I'm using Shortcake on (thanks so much btw!) and after I did, I am unable to see the actual shortcode content in the post editor. I just see the media loading graphic but it never actually loads. http://grab.by/GeZk

There don't appear to be any issues with actual functionality on the front end but I think my clients would be confused by this.

I downgraded back to v0.2.1 and it works correctly again. I did this on 3 separate sites as well. Thanks!

**Comment by chrisvanpatten** [comment]
I've noticed this as well. Flipping to the text editor and back to visual fixes it.

I upgraded the dev copy of my site from 4.1.1 to 4.2-trunk and that fixed it, so I'm guessing it's related to some of the 4.2 compat fixes. I'm running back through commits to see where it broke... I'll report back if I figure it out.

**Comment by chrisvanpatten** [comment]
So as far as I can tell the last commit that worked in 4.1.1 was 1575335fa623250c48523d46071d421b08d73e0e, which suggests to me that it's a problem in 9005e709454fd25d8346e445a5be07fca5f81267. Maybe those 'true's are required?

**Comment by danielbachhuber** [comment]
Dang. I didn't see this in my testing. Can you provide explicit reproduction steps?

**Comment by bpdeconi** [comment]
I'm experiencing this as well. Here's what I've experienced:

1) Add Media > Insert Post Element > Select your shortcode
2) Fill out the attributes, click Insert Element. Important: Don't click the Preview tab.
3) Result: Loading placeholder appears in the editor, but never resolves into the shortcode content.
4) Click the edit button your shortcode element, but don't make any changes.
5) Click Update.
6) Result: Shortcode element appears as desired in the editor.
7) Create a new shortcode.
8) Before Insert Element, click the Preview tab.
9) Preview displays as desired.
10) Click Insert Element.
11) Result: Shortcode element appears as desired in the editor.

In other words: I've only experienced this behavior when I create a new shortcode element and don't preview it first.

Hope this gets resolved in 0.2.3. This is a phenomenal addition to WP.

**Comment by danielbachhuber** [comment]
Hm, I can't reproduce using these steps. @mattheu's PR makes sense to me though.

**Comment by danielbachhuber** [comment]
@bpdeconi @chrisvanpatten Can you confirm the fix?

**Comment by bpdeconi** [comment]
@danielbachhuber Yes, this is fixed for me.

**Comment by hirejordansmith** [comment]
Working like a charm on every site I just updated to v0.2.3. Thanks so much for your work on this plugin. I've been looking for a reliable shortcode UI for some time.

**Comment by danielbachhuber** [comment]
Glad to hear! Thanks for following up.

**Comment by chrisvanpatten** [comment]
Can confirm it's all good for me too. Thanks! :rocket:
